ocs_ci/helpers/helpers.py

```
@@ -7005,16 +7005,28 @@ def find_cephblockpoolradosnamespace(storageclient_uid=None):
+    # If there are custom pools there will be multiple radosnamespace for internal storageconsumer and
+    # remote storageconsumers. The default interal cephblockpoolradosnamespace will start with
+    # ocs-storagecluster-cephblockpool or replicated-metadata-pool
     if storageconsumer == constants.INTERNAL_STORAGE_CONSUMER_NAME:
         cephbpradosns = list(
-            filter(lambda x: "-builtin-implicit" in x, cephblockpool_rns_names)
+            filter(
+                lambda x: "replicated-metadata-pool-builtin-implicit" in x
+                or "ocs-storagecluster-cephblockpool-builtin-implicit" in x,
+                cephblockpool_rns_names,
+            )
         )[0]
```

Ruff (0.15.17)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)